public/pages/AnomalyCharts/containers/AnomalyDetailsChart.tsx

```diff
@@ -391,7 +402,7 @@ export const AnomalyDetailsChart = React.memo(
               {
               coordinates: {
                 x0: anomaly.startTime,
-                x1: anomaly.endTime,
+                x1: anomaly.endTime + (anomaly.endTime - anomaly.startTime),
```

Comment:
what is this change for?

Reply:
make the rectangle stay at the center of the start time and end time